contracts/test/integration/GatewayDiamond.t.sol

```diff
@@ -1161,9 +1140,9 @@ contract GatewayActorDiamondTest is Test, IntegrationTestBase {
 
         IpcEnvelope[] memory msgs = new IpcEnvelope[](10);
         for (uint64 i = 0; i < 10; i++) {
-            msgs[i] = TestUtils.newTransferCrossMsg(
+            msgs[i] = TestUtils.newXnetCallMsg(
                 IPCAddress({subnetId: subnetId, rawAddress: FvmAddressHelper.from(caller)}),
-                IPCAddress({subnetId: gwGetter.getNetworkName(), rawAddress: FvmAddressHelper.from(caller)}),
+                IPCAddress({subnetId: gwGetter.getNetworkName(), rawAddress: FvmAddressHelper.from(vm.addr(100 + i))}),
```

Comment:
This change was needed because we were previously transferring to the subnet actor. We are now _calling_ instead of _transferring_, which calls the target address on the `handleIpcCall` method. Since the subnet actor didn't have that method, it failed. We now call an Eth account, and everyone is happy.